OverlayPlugin.Core/resources/opcodes.jsonc

```diff
@@ -81,6 +81,14 @@
             "ActorSetPos": {
                 "opcode": 570,
                 "size": 24
+            },
+            "SpawnObject": {
+                "opcode": 153,
+                "size": 64
+            },
+            "DespawnObject": {
+                "opcode": 956,
+                "size": 8
```

Comment:
Guessing opcodes probably need an update for 7.05 due to my negligent delay in re-reviewing?